### PR TITLE
change stride kernel(3,1) to stride kernel(2,2)

### PR DESCRIPTION
**Pull request by fanlu**
Hi, @csukuangfj, please review this pr.

### CODE REVIEW

File: egs/aishell/s10/chain/tdnnf_layer.py [hunk 1]
```diff
@@ -213,10 +206,10 @@ def forward(self, x):
 
         # TODO(fangjun): implement GeneralDropoutComponent in PyTorch
 
-        if self.linear.kernel_size == 3:
-            x = self.bypass_scale * input_x[:, :, 1:-1:self.conv_stride] + x
+        if self.linear.kernel_size == 2:
+            x = self.bypass_scale * input_x[:, :, self.subsampling_factor:-self.subsampling_factor:self.subsampling_factor] + x
```
Comment (danpovey) [inline egs/aishell/s10/chain/tdnnf_layer.py]: to make this line easier to read I'd use a variable `s = self.subsampling_factor`, defined just before this.

**Comment by danpovey** [comment]
Looks fine.  don't forget to put `affine=False` for all batchnorms.   
I'm curious what the results are.

File: egs/aishell/s10/chain/tdnnf_layer.py (hunk 1 above)
Comment (fanlu) [reply egs/aishell/s10/chain/tdnnf_layer.py]: OK

**Comment by fanlu** [comment]
the result have already posted in the pr https://github.com/kaldi-asr/kaldi/pull/3923
| | original stride kernel(3,1) | stride kernel(2,2) | 
|-- | -- | -- |
|dev_cer | 7.36 | 7.35 | 
|dev_wer | 15.55 | 15.48 | 
|test_cer | 9.25 | 9.08 | 
|test_wer | 18.08 | 17.85 | 

@csukuangfj would you like to make this result double check?

File: egs/aishell/s10/chain/tdnnf_layer.py [hunk 2]
```diff
@@ -194,13 +187,13 @@ def forward(self, x):
 
         # save it for skip connection
         input_x = x
-
+        logging.debug(f"input_x shape is {input_x.shape}")
```
Comment (csukuangfj) [inline egs/aishell/s10/chain/tdnnf_layer.py:190]: I think f-style string is only available in python 3.6.

Could you please use `"".format()` instead.

**Comment by csukuangfj** [comment]
@fanlu 
thank you very much.

File: egs/aishell/s10/chain/tdnnf_layer.py (hunk 2 above)
Comment (csukuangfj) [reply egs/aishell/s10/chain/tdnnf_layer.py:190]: thanks, looks good to me, merging.

**Pull request merged**